distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -411,14 +404,6 @@ type File
     last_modified_time self =
         handle_java_exceptions self <| self.last_modified_time_builtin
 
-    ## PRIVATE
-
-       Builtin method that gets this file's last modified time.
-       Recommended to use `File.last_modified_time` instead which handles
-       potential exceptions.
-    last_modified_time_builtin : ZonedDateTime
-    last_modified_time_builtin self = @Builtin_Method "File.last_modified_time_builtin"
-
```

Comment:
Why are we removing these?

IMO it is worth keeping them to document that the builtin exists and what signature it expects. Of course that is an internal thing, but it makes checking stuff much easier than having to look for the builtin only in Java code.

Reply:
So personally I don't find them particularly useful since they are essentially repeating what is defined by the public wrapper.
And the wrapper already has type info (+ potential exception). In fact, this duplication already seems to have led to some disparity because some of those don't agree on their return type even though they should.

So to me it seemed like the cost of maintaining those was just unnecessary. But I can bring it back if you disagree.

Reply:
You have convinced me :slightly_smiling_face: